install_functions/checks.sh

```diff
@@ -477,6 +477,34 @@ function checks_filebeatURL() {
     fi
 }
 
+function checks_development_source_tag() {
+    last_stage=""
```

Comment:
This variable should be in the `installVariables.sh` along with the source_branch